### PR TITLE
node-perf/tf-wide-deep: add support for Arm64

### PR DESCRIPTION
**Pull request by lubinsz**
Signed-off-by: Bin Lu <bin.lu@arm.com>

**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind failing-test

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:
node-perf/tf-wide-deep: add support for Arm64

**Does this PR introduce a user-facing change?**:
NONE
```release-note
NONE
```

### CODE REVIEW

**Review by neolit123 — COMMENTED** [review]
/ok-to-test
/kind feature
/priority backlog

File: test/images/node-perf/tf-wide-deep/Dockerfile [hunk 1]
```diff
@@ -14,8 +14,15 @@
 
 FROM BASEIMAGE
 
+CROSS_BUILD_COPY qemu-QEMUARCH-static /usr/bin/
+
 RUN apt-get update && apt-get install -y wget time
 RUN pip install tensorflow
```
Comment (mkumatag) [inline test/images/node-perf/tf-wide-deep/Dockerfile]: you need to remove this line as you are adding following if/else block for individual archs.

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: test/images/node-perf/tf-wide-deep/Dockerfile (hunk 1 above)
Comment (lubinsz) [reply test/images/node-perf/tf-wide-deep/Dockerfile]: @mkumatag 
Hi,
In my opinion, we are not register the 'F' flag for 'binfmt_misc', and so we need `qemu-static` inside the container.
If without this line. The building will be failed.
Please see following as reference:

make all WHAT=node-perf/tf-wide-deep
...
.,.
Step 1/7 : FROM arm64v8/python:3.6-slim-stretch
3.6-slim-stretch: Pulling from arm64v8/python
Digest: sha256:bd9b8e8f25e7eafdd1c09583d57bb0362645ea294f01c714b88f486931a9cf45
Status: Image is up to date for arm64v8/python:3.6-slim-stretch
 ---> 8e3c7c26d38b
Step 2/7 : RUN apt-get update && apt-get install -y wget time
 ---> Running in afe74ccd5e2d
standard_init_linux.go:190: exec user process caused "no such file or directory"
The command '/bin/sh -c apt-get update && apt-get install -y wget time' returned a non-zero code: 1
Makefile:37: recipe for target 'all-container' failed
make: *** [all-container] Error 1

Comment (mkumatag) [reply test/images/node-perf/tf-wide-deep/Dockerfile]: I'm talking about `RUN pip install tensorflow` line

Comment (lubinsz) [reply test/images/node-perf/tf-wide-deep/Dockerfile]: @mkumatag 
Thank you very much.
I have removed it. Please check.

**Comment by lubinsz** [comment]
@mkumatag 
Any comments?

File: test/images/node-perf/tf-wide-deep/BASEIMAGE [hunk 2]
```diff
@@ -1 +1,2 @@
 amd64=python:3.6-slim-stretch
+arm64=arm64v8/python:3.6-slim-stretch
```
Comment (ixdy) [inline test/images/node-perf/tf-wide-deep/BASEIMAGE]: do we want to add other arches too?

Comment (lubinsz) [reply test/images/node-perf/tf-wide-deep/BASEIMAGE]: I have tried it, and I have not found the required tensorflow package on other non-amd64 platforms. Such as, I tried to install tensorflow or tensorflow-ppc64le on ppc64le platform, but  failed.

Comment (mkumatag) [reply test/images/node-perf/tf-wide-deep/BASEIMAGE]: for ppc64le we do have ppa published but its not that straight forward. I will work with the concerned team in IBM and push that PR later.

**Comment by lubinsz** [comment]
@BenTheElder @mkumatag 
Can we merge it?

**Review by BenTheElder — APPROVED** [review]
/lgtm

**Comment by lubinsz** [comment]
@BenTheElder 
Can we merge it into milestone v1.14?

**Comment by BenTheElder** [comment]
given how close to release we are, changes that aren't bug fixes or already approved should probably go into early 1.15 soon.

**Pull request merged**